tests/ui/conftest.py

```diff
@@ -2,10 +2,24 @@
 import os
 import urlparse
 
+from waffle.testutils import override_switch
+from waffle.models import Flag
 from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
+from pytest_django import live_server_helper
 import jwt
 import pytest
 import requests
+import json
+
+from django.core.management import call_command
+
+from olympia.amo.tests import create_switch
+from olympia.access.models import Group
+
+
+@pytest.fixture
+def our_base_url(our_live_server):
+    return our_live_server.url
```

Comment:
Why aren't we using `base_url`? We should be able to do something like:

```python
@pytest.fixture(scope='session')
def base_url(base_url, request):
    return base_url or request.getfuncargvalue("live_server").url
```

This allows a base URL to be set, but if not, uses the live server.